<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>840</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1120</width>
     <height>470</height>
    </size>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>841</width>
      <height>485</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>36</number>
     </property>
     <property name="leftMargin">
      <number>36</number>
     </property>
     <property name="topMargin">
      <number>18</number>
     </property>
     <property name="rightMargin">
      <number>36</number>
     </property>
     <property name="bottomMargin">
      <number>18</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="image1">
         <property name="minimumSize">
          <size>
           <width>260</width>
           <height>260</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>sudoku-game-2.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="openExternalLinks">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>12</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Sudoku Game</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <property name="horizontalSpacing">
          <number>24</number>
         </property>
         <property name="verticalSpacing">
          <number>16</number>
         </property>
         <item row="2" column="1">
          <widget class="QPushButton" name="playButton">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Play</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="difficultyLabel">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Game difficulty</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="boardLabel">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Board</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="boardComboBox">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>Generate</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Load from file</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="difficultyComboBox">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <item>
            <property name="text">
             <string>Dev</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Easy</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Normal</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Hard</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="errorsCheckBox">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Realitme errors</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="image2">
         <property name="minimumSize">
          <size>
           <width>260</width>
           <height>260</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>sudoku-solver-1.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="openExternalLinks">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>12</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferDefault</stylestrategy>
          </font>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Sudoku Solver</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0">
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <property name="horizontalSpacing">
          <number>24</number>
         </property>
         <property name="verticalSpacing">
          <number>16</number>
         </property>
         <item row="0" column="1">
          <widget class="QPushButton" name="tryButton">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Try</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>76</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>842</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
